src/types/mod.rs

```diff
@@ -612,6 +612,8 @@ pub enum SpecialTransactionOutcome {
         development_charge: Amount,
     },
     /// Payment for a particular account.
+    /// The account delegates to the pool given by the last PaydayPoolReward
+    /// outcome listed before this outcome.
```

Comment:
This comment is not really in the correct place. This variant on its own has no notion of "last outcome before". That is only given meaning when these outcomes are part of the block summary.

Reply:
Done.